### PR TITLE
fix: placeholder vertical alignment in safari

### PR DESCRIPTION
**Pull request by rengare**
## Description
PR fixes placeholder issue in safari

### Before:
image

### After:
image

### CODE REVIEW

File: src/input.scss [hunk 1]
```diff
@@ -32,11 +32,12 @@ $block: #{$fd-namespace}-input;
   }
 
   &::placeholder {
-    @include fd-reset();
-
     line-height: $fd-form-input-height;
     font-style: italic;
-    color: var(--sapField_PlaceholderTextColor);
```
Comment (JKMarkowski) [inline src/input.scss]: Do we wan't to remove `color` from the placeholder?

Comment (stefanoScalzo) [reply src/input.scss]: No we don't

Comment (InnaAtanasova) [reply src/input.scss]: Actually we do! The color of the placeholder is set in _forms.scss

Comment (InnaAtanasova) [reply src/input.scss]: font-style is also not necessary

**Comment by estelaxu3** [comment]
I think the text is not aligned properly, it should move down 1 or 2 pixel. 
Screen Shot 2019-12-16 at 11 46 17 AM

File: src/input.scss (hunk 1 above)
Comment (rengare) [reply src/input.scss]: right, removed by mistake, revert is comming

**Comment by InnaAtanasova** [comment]
I opened a PR just to be able to test the Netlify deploy in IE11 but if you remove all `placeholder` rules in `src/input.scss` everything is fixed. There is no need of setting the line-height in either cozy or compact mode

https://github.com/SAP/fundamental-styles/pull/521/files

**Comment by rengare** [comment]
fixed provided by #521

**Pull request closed without merging**